app/assets/stylesheets/application.sass

```diff
@@ -4,3 +4,4 @@
 @import "buttons"
 @import "sortable"
 @import "admin"
+@import "dashboard"
```

Comment:
so is this the github dashboard css

Reply:
yep just so i could style the link as a button